src/legacy/core_plugins/kibana/public/discover/doc_table/doc_table.js

```diff
@@ -29,8 +29,7 @@ import './lib/pager';
 
 import { getLimitedSearchResultsMessage } from './doc_table_strings';
 
-uiModules
-  .get('app/discover')
+uiModules.get('app/discover')
```

Comment:
@mattkime  I reverted to old code. But according to [the JavaScript Styleguide](https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md), it is recommended to use prettier. I guess that the our goal is to change the entire code base with prettier like the previous version of this commit. 

Later, I should break the commit when there are changes by prettier to help others distinguish what's done by me and what's done by prettier. 

If you're new to prettier, you can learn [how to set up in this article](https://www.robinwieruch.de/how-to-use-prettier-vscode). You'll soon wonder how you wrote code without this. 

Note. Currently, prettier removes space between `function (`. We need to fix it manually with `node scripts/pre_commit --fix`. It seems that it can be fixed with [prettier-eslint](https://github.com/prettier/prettier-eslint). I need to raise an issue about this.